design/ceph/object/ceph-rgw-k8s-sa-authentication.md

```diff
@@ -0,0 +1,182 @@
+---
+Service authentication with vault for RGW
+target-version: release-1.15
```

Comment:
Does #13934 already implement the complete design? Or how much is left? v1.15 is just in the next couple weeks, this seems too soon since it hasn't been reviewed yet.